### ISSUE TITLE
Upload cancels not working for MultiUploader examples in IE7

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#18]
```
What steps will reproduce the problem?
1. Open the following url in an IE7 browser:
http://gwtupload.alcala.org/gupld/gwtuploadsample.MultipleUploadSample/Mult
ipleUploadSample.html
2. Queue up two files for upload using any of the 3 MultiUploader examples
3. Attempt to cancel the first file upload while it is in progress (this 
will sometimes fail)
4. If the first cancel succeeds, attempt to cancel the second file upload 
while it is in progress (this will always fail in IE7, but I have not seen 
this issue in Chrome or Firefox)

What is the expected output? What do you see instead?
I expect the upload to cancel. When the issue occurs, the UI just shows 
that the job is cancelling and just hangs there. The upload never cancels 
and the uploader never moves to the next upload job. I was able to 
reproduce this issue while running the examples in eclipe using the google 
app engine plugin. It appears that the cancel call back never makes it 
back to the server, and never gets logged when this issue occurs. The 
upload actually continues and completes even though the UI shows 
cancelling. 


What version of the product are you using? On what operating system?
I'm using the lastest GWTUpload project from the SVN repository. I'm 
running Windows XP Professional SP2. 


Please provide any additional information below.


```

Original issue reported on code.google.com by `john.m.b...@gmail.com` on 27 Oct 2009 at 5:08

**Comment by GoogleCodeExporter** [comment]
```
Hi, I've reproduced the problem and I'm trying to debug it in a windows machine.

Thanks.
Manolo
```

Original comment by `manuel.carrasco.m` on 30 Oct 2009 at 12:06
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 30 Oct 2009 at 12:18

**Comment by GoogleCodeExporter** [comment]
```
Hi, 

I think i've fixed the problem. It seems a cache issue in IE with GET Ajax 
requests.
Could you check if it doesn't happen any more?.  You have to use the demo page
because I've not released a new version.

Thanks
Manolo
```

Original comment by `manuel.carrasco.m` on 30 Oct 2009 at 1:24
- Changed state: **Feedback**

**Comment by GoogleCodeExporter** [comment]
```
Hey Manolo,

The issue is fixed. Do you have a date planned for the release that will 
include this 
fix?

Thanks,

John
```

Original comment by `john.m.b...@gmail.com` on 2 Nov 2009 at 4:16

**Comment by GoogleCodeExporter** [comment]
```
No plans right now, but I think in two weeks it should be ready the version 0.6 
which
includes maven packaging.

In the meanwhile you can download the version 0.5.4 because I've updated the 
files
with the fix.

Thanks
Manolo
```

Original comment by `manuel.carrasco.m` on 2 Nov 2009 at 5:09

**Comment by GoogleCodeExporter** [comment]
```
Thanks a lot, the new 0.5.4 jar works perfectly in all browsers. 
```

Original comment by `john.m.b...@gmail.com` on 3 Nov 2009 at 5:32

**Comment by GoogleCodeExporter** [comment]
```
released version 0.5.5 which includes de fix
```

Original comment by `manuel.carrasco.m` on 12 Nov 2009 at 10:42
- Changed state: **Fixed**
